### PR TITLE
[Doc Improvement][Manage your apps in Developer Portal][3388812]

### PR DESCRIPTION
**Pull request by v-vanisreeth**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/manage-your-apps-in-developer-portal.md](https://github.com/MicrosoftDocs/msteams-docs/blob/5939d2842d99a37ecf8f07ec6e7fc50bdf688168/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fmanage-your-apps-in-developer-portal.md) | 94 | 14 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d3276827-c7ff-47bd-95ce-d1419622106a) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/manage-your-apps-in-developer-portal.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6762b3e355d1ddc08c295e976b19e8e351311721/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fmanage-your-apps-in-developer-portal.md) | 96 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d87fc537-f2cd-4ed8-b57a-0227dfc1f77d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/concepts/build-and-test/manage-your-apps-in-developer-portal.md [hunk 1]
```diff
@@ -169,6 +169,9 @@ To add a bot:
 
 From the Developer Portal, you can go to Bot Framework portal and configure your bot to update icon and other properties.
 
+  > [!NOTE]
+  > Environment variables are not supported in Bot id.
```
Comment (v-deabira) [inline msteams-platform/concepts/build-and-test/manage-your-apps-in-developer-portal.md:173]: >[!NOTE] 
>Bot ID doesn't support environment variables.

**Review by v-deabira — CHANGES_REQUESTED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/manage-your-apps-in-developer-portal.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c807e3713f849013a64c8dfc87400bfa787a7727/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fmanage-your-apps-in-developer-portal.md) | 96 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b44fb4a9-5b12-4827-9a59-36d5ce6c51d8) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-habibaumme — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/manage-your-apps-in-developer-portal.md](https://github.com/MicrosoftDocs/msteams-docs/blob/fdc9730a6948849f8bcc3cc6666f7df3cb6becfd/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fmanage-your-apps-in-developer-portal.md) | 96 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/08cc5d32-d4eb-4fe7-8e83-e7217e2cd5b4) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Review by Wajeed-msft — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/manage-your-apps-in-developer-portal.md](https://github.com/MicrosoftDocs/msteams-docs/blob/30157157e45f5924d1c2559fffa441b4c5f94d58/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fmanage-your-apps-in-developer-portal.md) | 96 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7b982ab9-cdb2-4aed-afd2-85c0fac3abaa) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Pull request merged**